modules/nextflow/src/main/groovy/nextflow/cli/CmdKubeRun.groovy

```diff
@@ -53,6 +53,12 @@ class CmdKubeRun extends CmdRun {
     @Parameter(names = '-head-memory', description = 'Specify amount of memory requested for the Nextflow pod')
     String headMemory
 
+    @Parameter(names=['-rc','-remoteConfig'], description = 'Choose configuration files on remote fs', hidden = true )
+    List<String> runRemoteConfig
```

Comment:
I would leave out the `-rc` shortcut just to keep things simple.

I would try to make the help text consistent with `-config`, e.g. "Add the specified file from the K8s cluster to configuration set"

Reply:
Good point. I addressed it.